plugins/field/views/switch.tsx

```diff
@@ -294,8 +309,8 @@ export function Body() {
       <div>
         <p className="py-4">
           <Translate>
-            You can apply rounded, colors and shapes to the 
-            <C l value="Switch" /> component.
+            You can use
+            the <C value="Switch" /> CSS class to globally theme the switch field.
           </Translate>
         </p>
```

Comment:
This entire statement is wrong. look at the `components/field/Switch.tsx` for css class names being used.